include/vere/vere.h

```diff
@@ -1201,12 +1201,12 @@
         u3_cttp_io_exit(void);
 
 
-      /* u3_dawn_come(): mine a comet under star (unit)
+      /* u3_dawn_come(): mine a comet
       */
         u3_noun
-        u3_dawn_come(u3_noun star);
+        u3_dawn_come(void);
 
-      /* u3_dawn_vent(): validatated boot event
+      /* u3_dawn_vent(): validated boot event
```

Comment:
Oof

Reply:
I mean, validatated is obviously a stronger acceptance criterion, but this is still relatively new code.